nix/launcher-config.nix

```diff
@@ -156,8 +163,11 @@ let
       network = {
         configFile = "configuration-${environment}.yaml";
         genesisFile = "${environment}-genesis.json";
-        genesisHash = envCfg.genesisHash;
+        genesisHash = if (environment != "selfnode") then envCfg.genesisHash else "";
```

Comment:
Remove from arguments @nikolaglumac

Reply:
This is not to be changed now as we are integrating against 1.8 version of the node which still requires the `genesisHash` so we will keep it as is.

cc @KtorZ @rhyslbw @rvl